src/dapps/ens-manager-dapp/TheENSManagerLayout.vue

```diff
@@ -39,7 +40,8 @@
                 @input="setName"
               />
               <mew-button
-                :disabled="name.length <= 0"
+                :loading="loading"
+                :disabled="(name && name.length === 0) || loading"
                 :has-full-width="false"
```

Comment:
can you make it full width and place button and input in columns? otherwise on loading button change's it's size

Reply:
done!